docker-compose.yml

```diff
@@ -1,35 +1,55 @@
+networks:
+ atomization-network: {}
+
 services:
-  nix:
-    image: nix
-    build:
-      dockerfile: ./docker/Dockerfile._nix
-  common:
-    image: common
-    build:
-      dockerfile: ./docker/Dockerfile.common
-    depends_on:
-      - nix
-  interactive:
-    build:
-      context: .
-      dockerfile: ./docker/Dockerfile.atomization
-    depends_on:
-      - common
-    volumes:
-      - ~/.ssh:/home/ubuntu/.ssh
-      - $PWD:/home/ubuntu
-    working_dir: /home/ubuntu
-    entrypoint: ["bash"]
-    stdin_open: true
-    tty: true
-  atomization:
-    build:
-      context: .
-      dockerfile: ./docker/Dockerfile.atomization
-    depends_on:
-      - common
-    volumes:
-      - ~/.ssh:/home/ubuntu/.ssh
-      - $PWD:/home/ubuntu
-    working_dir: /home/ubuntu
-    entrypoint: ["./docker/entrypoint.sh"]
+ nix:
+   image: nix
+   build:
+     dockerfile: ./docker/Dockerfile._nix
+   networks:
+     - atomization-network
+
+ common:
+   image: common
+   build:
+     dockerfile: ./docker/Dockerfile.common
+   depends_on:
+     - nix
+   networks:
+     - atomization-network
+
+ interactive:
+   build:
+     context: .
+     dockerfile: ./docker/Dockerfile.atomization
+   depends_on:
+     - common
+   volumes:
+     - ~/.ssh:/home/ubuntu/.ssh
+     - $PWD:/home/ubuntu
+   working_dir: /home/ubuntu
+   entrypoint: ["bash"]
+   stdin_open: true
+   tty: true
+   networks:
+     - atomization-network
+   extra_hosts:
+     - "host.docker.internal:host-gateway"
+
+ atomization:
+  networks:
+    - atomization-network
+  environment:
+    DB_HOST: 172.17.0.1
```

Comment:
I suspect it's better to put this in .env

Reply:
that's the IP for the docker container's reference to the HOST. Never should change.